File: website/src/app/models/travel.ts
import { City } from "./city";


export interface Travel {
    id:number;
    travelType:TravelType;
    date: Date;
    fromCity : City;
    toCity: City; //TODO: Remove 
}

export enum TravelType {
    Flight=0,
    Bus=1,
    Train=2,
    Boat=3,
    Scooter=4,
    Walk=5,
    Taxi=6
}

File: Api/Models/Travel.cs
namespace TravelPlannerApp.Models
{
    public enum TravelType
    {
        Flight,
        Bus,
        Train,
        Boat,
        Scooter,
        Walk,
        Taxi
    }
    public class Travel
    {
        public int Id { get; set; }
        public TravelType TravelType { get; set; }
        public DateTime Date { get; set; }
        public City FromCity { get; set; }
        public City? ToCity { get; set; }
    }
}
